[ADD] l10n_in: add information about PAN field in india (#10978)
odoo/documentation: content/applications/finance/fiscal_localizations/india.rst

.. _india/e-invoicing:

Indian Configuration
====================

In :menuselection:`Settings --> Users & Companies --> Companies`, add your :guilabel:`PAN` and

> :guilabel:`GSTIN`. The :guilabel:`PAN` is essential for determining the type of taxpayer,
> while :guilabel:`GSTIN`is required for generating e-Invoices and E-waybills.

Reviewer: ```suggestion
:guilabel:`GSTIN`. The PAN is essential for determining the type of taxpayer,
while GSTIN is required for generating e-Invoices and E-waybills.
```
Guilabels aren't necessary in this case, and the second guilabel caused a _ci/documentation_ error due to a missing space after the second backtick :)

Author: Hello @xpl-odoo 
made the suggested changes.

.. _india/e-invoicing:

Indian Configuration
====================

> In :menuselection:`Settings --> Users & Companies --> Companies`, add your PAN and GST number.

Reviewer 2: ```suggestion
In :menuselection:`Settings --> Users & Companies --> Companies`, add your :guilabel:`PAN` and
:guilabel:`GSTIN` to ensure the correct chart of accounts and taxes are installed.
```

Reviewer 2: We use guilabels when talking about an interactive element such as a field to complete, and I suggested a little more information to give the reader more context as to why it's important.

Author: > We use guilabels when talking about an interactive element such as a field to complete, and I suggested a little more information to give the reader more context as to why it's important.

Hello @dade-odoo , I have updated the information with correct reason for readers understanding.
thanks!!

> 
> .. image:: india/company-form.png
>    :alt: Company form

Reviewer 2: ```suggestion
```

Reviewer 2: We try to limit the number of screenshots and only include them when they're really necessary, so I suggest removing this one (don't forget to delete the actual file as well).
